src/aditiGJUN20/StringRotation.java

```diff
@@ -0,0 +1,44 @@
+/*Coding test 10 :
+
+Print minimum rotation required to compare both strings equality.
+
+Example 1  : 
+String input1 = "abcdef";
+String input2 = "defabc";
+output : 3
+
+Example 2  : 
+String input1 = "adbcdef"; -> push right
+fadbcde
+efadbcd
+defadbc
+String input2 = "defadbc";
+output : 3*/
+
+package aditiGJUN20;
+
+public class StringRotation {
+	int getStringRotationCount(String input1, String input2) {
+		int count = 0;
+		if (input1.equals(null) || input2.equals(null) || input1.length() != input2.length())
+			System.out.println("Strings are not matching with any rotation");
+		else {
+			while (!input1.equals(input2)) {
```

Comment:
Just a suggestion. what if String can't be made equals even after all possible rotation ?? this will run infinitely ...

you can handled it by adding one more condition.
while(!input1.equals(input2) && count< input1.length)